bin/git/hooks/pre-commit/validate-diffs

```diff
@@ -25,7 +25,3 @@ fi
 git diff --cached --name-only --diff-filter=A -z $against |
         LC_ALL=C tr -d '[ -~]\0' | wc -c
 check_rc "Rename non-ASCII file name(s) before committing"
-
-# If there are whitespace errors, print the offending file names and fail.
```

Comment:
Removed this because the NXAPI and gRPC tests include trailing whitespace as part of the expected literal output from the node. Rubocop will catch trailing whitespace in Ruby code so this doesn't give us a lot of risk.